src/main/java/nl/hsac/fitnesse/fixture/slim/EmailFixture.java

```diff
@@ -0,0 +1,210 @@
+        return getText(text, regex);
+    }
+
+    private String getText(final String text, final String regexpattern) {
+        LOGGER.debug("Text found: " + text);
+        LOGGER.debug("Regex: " + regexpattern);
+        final Pattern pattern = Pattern.compile(regexpattern);
+        final Matcher matcher = pattern.matcher(text);
+        final boolean isMatch = matcher.find();
+        if (isMatch) {
+            return matcher.group(1);
+        }
+        throw new SlimFixtureException(false, format("No match found in text %s with regex %s", text, regexpattern));
+    }
+
+    private String getMailText(SearchParameters params) {
+        try {
+            final Folder inbox = getInboxFolder();
+            Message msg = getRecentMessagesMatching(inbox, params);
+            return getBody(msg);
+        } catch (final Exception ex) {
+            throw new SlimFixtureException(false, "No message found with search params: " + params.toString(), ex);
+        }
+    }
+
+    private Folder getInboxFolder() throws MessagingException {
+        final Folder inbox = store.getFolder("inbox");
+        inbox.open(Folder.READ_WRITE);
+        return inbox;
+    }
+
+    private Message getRecentMessagesMatching(Folder inbox, SearchParameters params) {
+        List<Message> mails = getMessagesMatchingAndWaitUntil(inbox, params);
+        Collections.reverse(mails);
+        return getMostRecentMessage(mails);
+    }
+
+    private Message getMostRecentMessage(List<Message> mails) {
+        try {
+            for (final Message mail : mails) {
+                if (mail.getSentDate().after(HOURS_BACK)) {
+                    return mail;
+                }
+            }
+        } catch (MessagingException ex) {
+            throw new RuntimeMessageException(ex);
+        }
+        throw new RuntimeIOException();
+    }
+
+    private List<Message> getMessagesMatchingAndWaitUntil(Folder inbox, SearchParameters params) {
+        FetchEmailsStrategy fetchEmails = new FetchEmailsStrategy(inbox, params);
+        boolean pretty = repeatUntil(fetchEmails);
+        if (!pretty)
+            throw new CouldNotFindMessageException(inbox, params);
+        return fetchEmails.getResult();
+    }
+
+    private class FetchEmailsStrategy implements RepeatCompletion {
+        private final Folder inbox;
+        private final SearchParameters params;
+        private List<Message> mails;
+
+        FetchEmailsStrategy(Folder inbox, SearchParameters params) {
+            this.inbox = inbox;
+            this.params = params;
+        }
+
+        public List<Message> getResult() {
+            return mails;
+        }
+
+        @Override
+        public boolean isFinished() {
+            return mails != null && mails.size() > 0;
+        }
+
+        @Override
+        public void repeat() {
+            try {
+                mails = getMessagesMatching(inbox, params);
+            } catch (RuntimeException ex) {
+            }
```

Comment:
Shouldn't th exception at least be logged (at debug or something). 
If ignoring is OK, please provide comment why.